test/pjrt/test_runtime_multi_cpu.py

```diff
@@ -41,17 +41,8 @@ def test_multi_cpu_devices(self):
     devices_per_process = pjrt.run_multiprocess(xm.xla_device)
     self.assertDictEqual(devices_per_process, expected)
 
-  @parameterized.named_parameters(('xla_model', xm.get_ordinal),
-                                  ('pjrt', xr.global_ordinal))
-  def test_global_ordinal(self, ordinal_func):
-    results = pjrt.run_multiprocess(ordinal_func)
-    self.assertListEqual(sorted(results.values()), [0, 1, 2, 3])
-
-  @parameterized.named_parameters(('xla_model', xm.get_local_ordinal),
-                                  ('pjrt', xr.local_ordinal))
-  def test_local_ordinal(self, ordinal_func):
-    # TODO(wcromar): add multiprocess tests
-    results = pjrt.run_multiprocess(ordinal_func)
+  def test_global_ordinal(self):
```

Comment:
why `test_global_ordinal ` is removed?

Reply:
`xm.get_ordinal` and `xr.global_ordinal` are the same function now. I think you mean `test_local_ordinal`? I may make a mistake here. Let me bring back `xr.local_ordinal` test.